scriptExamples/extendedMode/system-tray/scriptable_systemtrayicon.py

```diff
@@ -0,0 +1,765 @@
+        "(255, 0, 0, 127)" for a semi-transparent red. These colors are used to
+        replace the fill color (`PostHorn._fill`) and the outline color
+        (`PostHorn._outline`).
+
+Icon
+====
+
+The icon included is a post horn. This can be changed by adapting the `_b64`
+data property in the `PostHorn` class:
+    1. Make a .png image and save it into a `pngfile`.
+    2. Encode it:
+          >>> base64.encodestring(open(pngfile,"rb").read())
+    3. Copy it and paste it into an editor.
+    4. Replace all "\n" with a newline.
+    5. Copy the string and paste it into `_b64 = '''<png-data>'''`
+
+The included post horn has a transparent background, a black outline and a
+white foreground. This is replaced with different shades of yellow, which are
+configured in the `CONFIG['color']` dictionary. You might want to keep the same
+scheme (transparent background, black outline, white foreground).
+
+
+TODO
+====
+
+- Add menu items, so that Thunderbird opens with the correct mail folder.
+- Add action(s) to the notification.
+
+
+Created on Thu Oct  6 19:49:17 2022
+
+Copyright (C) 2022  Stephan Helma
+
+"""
+
+import base64
+import io
+import json
+import os
+import pathlib
+import signal
+import struct
+import sys
+import time
+
+from PIL import Image
+import pystray
+
+
+__version_info__ = (0, 1, 0)
+__version__ = '.'.join([str(n) for n in __version_info__])
+__pyversion__ = '>=3.6'             # We use f-strings!
+
+
+# The configuration
+CONFIG = {
+    'notify': False,                # Show notifications
+    'show_read_icon': False,        # Show icon, when all messages are read
+    'mail': 'thunderbird --mail',   # Menu item "Thunderbird"
+    'mail_reset': 'thunderbird --mail',     # Menu item "Thunderbird/Reset"
+    'reset_menu': True,             # Menu item "Reset message counter"
+    'quit_menu': False,             # Menu item "Quit"
+    'color': {
+        'new mail': {
+            'fill': (238, 210, 2, 255),         # safety yellow
+            'outline': (255, 170, 29, 255)      # bright yellow
+        },
+        'some mail': {
+            'fill': (0, 0, 0, 0),               # transparent
+            'outline': (255, 170, 29, 255)      # bright yellow
+        },
+        'no mail': {
+            'fill': (255, 255, 255, 255),       # white
+            'outline': (127, 127, 127, 255)     # gray
+        },
+        'indifferent': {
+            'fill': (0, 0, 0, 0),               # transparent
+            'outline': (255, 255, 255, 191)     # white
+        }
+    },
+#    'logging': 'WARNING'           # Logging level, set to `None` to disable it
+#    'logging': 'INFO'
+#    'logging': 'DEBUG'
+    'logging': None
+}
+
+
+# The logging system
+if CONFIG['logging']:
+    import logging
+    logging.basicConfig(
+        level=getattr(logging, CONFIG['logging']),
+        format='%(levelname)8s %(funcName)s%(message)s',
+        filename=f'/home/sph/{pathlib.Path(__file__).with_suffix(".log").name}')
```

Comment:
Shouldn't this path be a variable?

Reply:
Oops, you are absolutely right!

Easy to change, but do we need logging at all?

Reply:
To be honest, you decide what's in your script. So as you wish!

Reply:
Done.